GoogleForCreators/web-stories-wp
# Dashboard: Add a save button to the google analytics id form

## Summary

Add a save button to the google analytics id form

![no-change](https://user-images.githubusercontent.com/2755722/92179488-bd3dfe00-ee12-11ea-8918-28b5ce9ffbdf.png)

![change](https://user-images.githubusercontent.com/2755722/92179514-cb8c1a00-ee12-11ea-89c3-8780806af286.png)



## User-facing changes

- Adds a button to the google analytics id form that becomes disabled under two exclusive conditions:
1. You supply an invalid GA id
2. You type in the currently saved GA id

## Testing Instructions
- Navigate to Editor Settings
- Type in a valid GA id (ua-#-#) and click save. Refresh the page and see that the GA id you entered is the value in the text box
- Edit the GA id to another valid GA id and press Enter.  Refresh the page and see that the GA id you entered is the value in the text box
- Delete the value from Input box and press Enter/Click Save.  Refresh the page to see that there is no GA id.

---

 #3843
